src/sentry/issues/auto_source_code_config/stacktraces.py

```diff
@@ -1,14 +1,18 @@
 import logging
-from collections.abc import Mapping
-from typing import Any
+from collections.abc import Mapping, Sequence
+from typing import Any, TypedDict
 
 from sentry.db.models.fields.node import NodeData
 from sentry.utils.safe import get_path
 
 logger = logging.getLogger(__name__)
 
 
-def identify_stacktrace_paths(data: NodeData) -> list[str]:
+class Stacktrace(TypedDict):
+    stacktrace: Mapping[str, Sequence[Mapping[str, Any]]]
+
+
+def identify_stacktrace_paths(data: NodeData | Stacktrace) -> list[str]:
```

Comment:
I'm writing some new tests I don't need to create a `NodeData` but just a stack trace.